popup.js

```diff
@@ -0,0 +1,135 @@
+const popUps = [
+  {
+    id: 'project-button1',
+    title: 'Multi-Post Stories',
+    image: './images/SnapshootPortfolio.png',
+    description:
+    "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown.",
+    technologies: ['Css', 'Html', 'Bootstrap'],
+    live_link: 'https://promzzykoncepts.github.io/My-Portfolio/',
+    source_link: 'https://github.com/promzzykoncepts/my-Portfolio/',
+  },
+
+  {
+    id: 'project-button2',
+    title: 'Professional Art Printing Data',
+    image: './images/SnapshootPortfolio.png',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown",
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    live_link: 'https://promzzykoncepts.github.io/My-Portfolio/',
+    source_link: 'https://github.com/promzzykoncepts/my-Portfolio/',
+  },
+  {
```

Comment:
- [ ] It would be nice if your popup contains the information for each project card i.e each project popup window should inherit the `image`, `technologies`, `description`, and `links`, from its unique card. Actually, you created the array that holds popup information but they have the same card information. Please ensure to implement these changes because adding projects in the future will be easy. You will only add it in one place - as another object in the projects array.

- [ ] Your popup looks nice but there are a few things that miss to make it match the Figma template popup. You will maybe notice that, on the Desktop screen, the links `buttons` icons disappear and I'm able to only see the popup for the first two cards - **This looks nice for a mobile screen**. Kindly make sure you add both icons and to other cards the `see project` buttons so the users can see other projects' popups.

- [ ] Following the Figma design the popup window has a `blur` background with a little opacity. KIndly add it also and remove any `border-radius` as provided in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471).

**Images: Your popup**
![Screenshot from 2022-07-19 23-26-55](https://user-images.githubusercontent.com/80612925/179851575-84c999f4-7b30-4e44-a401-bc3c557bfc9d.png)

image: The Figma popup

![Screenshot from 2022-07-19 23-27-21](https://user-images.githubusercontent.com/80612925/179851607-d357df7f-ffc3-415f-ba43-56c8265b6d5d.png)